### PR TITLE
Create de_DE.po

### PR DESCRIPTION
**Pull request by Etamuk**

### CODE REVIEW

File: de_DE.po [hunk 1]
```diff
@@ -0,0 +1,1085 @@
...
+"?"
+msgstr ""
+"Fehler beim Öffnen dieser Datei. Meinen Sie\n"
+"flatpak run --file-forwarding {0} @@ {1} @@\n"
+"?"
+
+#. To tranlators: "translate" this by your name, it will be displayed in the "about" dialog
+#: src/main.py:269
+msgid "translator-credits"
+msgstr "Dank an Übersetzer"
```
Comment (maoschanz) [inline de_DE.po]: This should be your name

Comment (maoschanz) [reply de_DE.po]: (or pseudonym)

Comment (Etamuk) [reply de_DE.po]: Done

**Comment by maoschanz** [comment]
oh fuck why wasn't it in /po/

**Comment by Etamuk** [comment]
Sorry. Seems i uploaded the file in the wrong directory. I'm not using Github often.

**Comment by maoschanz** [comment]
using the git command is often the simplest way to do

**Pull request merged**